website/docs/release_notes.md

```diff
@@ -7,6 +7,127 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.1](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.1)
+
+### Add a guide for migrating from the old `scala` runner to Scala CLI
+
+As of [SIP-46](https://github.com/scala/improvement-proposals/pull/46), Scala CLI has been accepted as the new `scala`
+command. To make the transition smooth we added a [guide](docs/guides/old-runner-migration.md) highlighting
+the differences between the two runners.
+
+Added by [@Gedochao](https://github.com/Gedochao)  in [#1900](https://github.com/VirtusLab/scala-cli/pull/1900)
+
+### Improve the `publish` and `publish setup` user experience
+
+We're currently focusing on improving the experimental `publish` feature of Scala CLI and making the UX of `publish setup` + `publish` better.
+
+Addressed by [@alexarchambault](https://github.com/alexarchambault) in [#1432](https://github.com/VirtusLab/scala-cli/pull/1432)
+and by [@MaciejG604](https://github.com/MaciejG604) in [#1898](https://github.com/VirtusLab/scala-cli/pull/1898)
+
+### Remove unsupported kebab-case style in using directives
+
+All using directives names are now using camelCase, kebab-case is no longer available.
+
+Added by [@lwronski](https://github.com/lwronski) in [#1878](https://github.com/VirtusLab/scala-cli/pull/1878)
+
+### Add a reference for available config keys in help & docs
+
+You can now view the available config keys using `config --help`:
+```
+scala-cli config -h
+# Usage: scala-cli config [options]
+# Configure global settings for Scala CLI.
+# 
+# Available keys:
+#   actions                                        Globally enables actionable diagnostics. Enabled by default.
+#   interactive                                    Globally enables interactive mode (the '--interactive' flag).
+#   power                                          Globally enables power mode (the '--power' launcher flag).
+#   suppress-warning.directives-in-multiple-files  Globally suppresses warnings about directives declared in multiple source files.
+#   suppress-warning.outdated-dependencies-files   Globally suppresses warnings about outdated dependencies.
+# 
+# You are currently viewing the basic help for the config sub-command. You can view the full help by running: 
+#    scala-cli config --help-full
+# For detailed documentation refer to our website: https://scala-cli.virtuslab.org/docs/commands/misc/config
+# 
+# Config options:
+#   --unset, --remove  Remove an entry from config
+```
+Also, `config --full-help` will show the list of all keys, including the ones used in commands that require `--power`.
```

Comment:
```suggestion
Also, `config --full-help` will show the list of all keys.
```
`--power` is actually irrelevant here and will be addressed separately in a future PR.